src/pytorch/neighbors/__init__.py

```diff
@@ -1,5 +1,10 @@
 '''
 Neighbor operations
 '''
+import site
+import os
+import torch
 
-from NNPOps.neighbors.getNeighborPairs import getNeighborPairs
\ No newline at end of file
+torch.ops.load_library(os.path.join(site.getsitepackages()[-1],"NNPOps", "libNNPOpsPyTorch.so"))
```

Comment:
Does this really need? I think `src/pytorch/__init__.py` is executed when `NNPOps.neighbors` is imported.

Reply:
This example does not run for me unless I load the library there:
```python
import torch as pt                                                                                                                                                           
import numpy as np                                                                                                                                                           
import pytest                                                                                                                                                                
from NNPOps.neighbors import getNeighborPairs                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
device = 'cuda:0'
dtype = pt.float32
num_atoms = 100
# Generate random positions
positions = 10 * pt.randn((num_atoms, 3), device=device, dtype=dtype)
cutoff = float(5.0)
num_neighbors = num_atoms*num_atoms
neighbors, deltas, distances = getNeighborPairs(positions, cutoff=cutoff, max_num_neighbors=num_neighbors)
```
Seems like if one includes NNPOps.neighbors directly as above the pytorch/__init__.py is not loaded.

Reply:
Then we should be able to remove the torch.load in neighbors. Lets see.